platform/ios/app/MBXViewController.m

```diff
@@ -1274,12 +1274,7 @@ - (void)styleRasterSource
 
 -(void)styleCountryLabelsLanguage
 {
-    NSArray<NSString *> *labelLayers = @[
-        @"country-label-lg",
-        @"country-label-md",
-        @"country-label-sm",
-    ];
-    [self styleLabelLanguageForLayersNamed:labelLayers];
+    [self.mapView.style setLocalizesLabels:YES];
```

Comment:
This method should account for `_usingLocaleBasedCountryLabels`. Otherwise, the “Label Countries in Local Language” option has no effect. Also, we should rename the option to “Show Labels in <language name or Local Language>”, because `MGLStyle.localizesLabels` localizes everything, not just country labels.